webui/react/src/components/UPlotChart/tooltipsPlugin.module.scss

```diff
@@ -0,0 +1,43 @@
+.tooltip {
```

Comment:
**nit:** Is it possible to move `Trial X` to be above `Time Elapsed (sec): X`?

<img width="959" alt="Screen Shot 2021-04-19 at 4 18 53 PM" src="https://user-images.githubusercontent.com/220971/115316136-ff177c80-a135-11eb-8b55-c6a3cd21ba16.png">

Reply:
**nit:** This is a weird one but the cursor dashed lines and the vertical cross line on the highlighted datapoint doesn't span all the way down and to the left of the charting area? The top and the right don't have the same problem.

![2021-04-19 at 10 56 PM](https://user-images.githubusercontent.com/220971/115339935-02752d00-a163-11eb-9ee1-49f70ca6729a.png)

Reply:
**> nit: Is it possible to move Trial X to be above Time Elapsed (sec): X?**

We should keep in mind that there's the possibility of multiple series with multiple labels to add (one for each value).
What do you think it's the best thing to do to accomodate both this requirements?

Reply:
(i'm gonna reply to the other question on another code line to start another thread).

Reply:
> What do you think it's the best thing to do to accomodate both this requirements?

Ah I see. Thoughts on supporting both cases (single series vs multi-series) a little differently?
Single Series:
```
[series label]
[x label: x data point]
(color) [y label: y data point]
```
Multi-Series:
```
[x-label: x data point]
(color1) [y1 series label]
              [y1 label: y1 data point]
(color2) [y2 series label]
              [y2 label: y2 data point]
(color3) [y3 series label]
              [y3 label: y3 data point]
...
```